diff --git a/mod/theme_bento/views/default/profile/menu/linksownpage.php b/mod/theme_bento/views/default/profile/menu/linksownpage.php
index 08aece0..057a578 100644
--- a/mod/theme_bento/views/default/profile/menu/linksownpage.php
+++ b/mod/theme_bento/views/default/profile/menu/linksownpage.php
@@ -28,6 +28,9 @@
 		<a href="<?php echo $vars['url']; ?>pg/friends/<?php echo $vars['entity']->username; ?>/"><?php echo elgg_echo("friends"); ?></a>	
 	</p>
 	<p class="user_menu_friends_of">
-		<a href="<?php echo $vars['url']; ?>pg/friendsof/<?php echo $vars['entity']->username; ?>/"><?php echo elgg_echo("friends:of"); ?></a>	
+		<a href="<?php echo $vars['url']; ?>pg/friendsof/<?php echo $vars['entity']->username; ?>/"><?php echo elgg_echo("friends:of"); ?></a>
+	</p>
+	<p class="user_menu_friends_of">
+		<a href="<?php echo $vars['url']; ?>pg/groups/member/<?php echo $vars['entity']->username; ?>/">Groups</a>
 	</p>
 	
\ No newline at end of file
