diff --git a/mod/event_calendar/start.php b/mod/event_calendar/start.php
index 5e68067..38936ab 100644
--- a/mod/event_calendar/start.php
+++ b/mod/event_calendar/start.php
@@ -103,12 +103,12 @@
 			if (!$site_calendar || $site_calendar == 'admin') {
 				if (isadminloggedin()) {
 					// only admins can post directly to the site-wide calendar
-					add_submenu_item(elgg_echo('event_calendar:new'), $CONFIG->url . "pg/event_calendar/new/", 'eventcalendaractions');
+					add_submenu_item(elgg_echo('event_calendar:new'), $CONFIG->url . "pg/event_calendar/new/");
 				}
 			} else if ($site_calendar == 'loggedin') {
 				// any logged-in user can post to the site calendar
 				if (isloggedin()) {
-					add_submenu_item(elgg_echo('event_calendar:new'), $CONFIG->url . "pg/event_calendar/new/", 'eventcalendaractions');
+					add_submenu_item(elgg_echo('event_calendar:new'), $CONFIG->url . "pg/event_calendar/new/");
 				}
 			}
 		}
@@ -117,9 +117,9 @@
 			if (($event_id = get_input('event_id',0)) && ($event = get_entity($event_id))) {
 				if (isadminloggedin() && (get_plugin_setting('allow_featured', 'event_calendar') == 'yes')) {
 					if ($event->featured) {
-						add_submenu_item(elgg_echo('event_calendar:unfeature'), $CONFIG->url . "action/event_calendar/unfeature?event_id=".$event_id.'&'.event_calendar_security_fields(), 'eventcalendaractions');
+						add_submenu_item(elgg_echo('event_calendar:unfeature'), $CONFIG->url . "action/event_calendar/unfeature?event_id=".$event_id.'&'.event_calendar_security_fields());
 					} else {
-						add_submenu_item(elgg_echo('event_calendar:feature'), $CONFIG->url . "action/event_calendar/feature?event_id=".$event_id.'&'.event_calendar_security_fields(), 'eventcalendaractions');
+						add_submenu_item(elgg_echo('event_calendar:feature'), $CONFIG->url . "action/event_calendar/feature?event_id=".$event_id.'&'.event_calendar_security_fields());
 					}
 				}
 				add_submenu_item(elgg_echo("event_calendar:view_link"), $CONFIG->wwwroot . "pg/event_calendar/view/" . $event_id,'0eventcalendaradmin');
@@ -187,4 +187,4 @@
 	register_action("event_calendar/add_to_group",false,$CONFIG->pluginspath . "event_calendar/actions/add_to_group.php");
 	register_action("event_calendar/remove_from_group",false,$CONFIG->pluginspath . "event_calendar/actions/remove_from_group.php");
 
-?>
\ No newline at end of file
+?>
