--- a/mod/profile/views/default/profile/edit.php
+++ b/mod/profile/views/default/profile/edit.php
@@ -35,6 +35,8 @@
 				$access_id = ACCESS_DEFAULT;
 			}
 
+			$admin_only = in_array($shortname, array('freenode_nick', 'freenode_cloak', 'email_nick', 'email_full', 'email_target'));
+			if (!$admin_only || isadminloggedin()) {
 ?>
 
 	<p>
@@ -51,7 +51,7 @@
 	</p>
 
 <?php
-
+			}
 		}
 
 ?>
--- a/mod/profile/actions/edit.php
+++ b/mod/profile/actions/edit.php
@@ -30,7 +30,7 @@
 			// the decoding is a stop gag to prevent &amp;&amp; showing up in profile fields
 			// because it is escaped on both input (get_input()) and output (view:output/text). see #561 and #1405.
 			// must decode in utf8 or string corruption occurs. see #1567.
-			$value = get_input($shortname);
+			$value = get_input($shortname, NULL);
 			if (is_null($value)) continue;
 			if (is_array($value)) {
 				array_walk_recursive($value, 'profile_array_decoder');
