diff --git a/mod/groups/start.php b/mod/groups/start.php
index a804274..b325b43 100644
--- a/mod/groups/start.php
+++ b/mod/groups/start.php
@@ -212,29 +212,29 @@
 			if ($page_owner instanceof ElggGroup && get_context() == 'groups') {
 				if (isloggedin()) {
 					if ($page_owner->canEdit()) {
-						add_submenu_item(elgg_echo('groups:edit'),$CONFIG->wwwroot . "pg/groups/edit/" . $page_owner->getGUID(), '1groupsactions');
-						add_submenu_item(elgg_echo('groups:invite'),$CONFIG->wwwroot . "pg/groups/invite/{$page_owner->getGUID()}", '1groupsactions');
+						add_submenu_item(elgg_echo('groups:edit'),$CONFIG->wwwroot . "pg/groups/edit/" . $page_owner->getGUID());
+						add_submenu_item(elgg_echo('groups:invite'),$CONFIG->wwwroot . "pg/groups/invite/{$page_owner->getGUID()}");
 						if (!$page_owner->isPublicMembership())
-							add_submenu_item(elgg_echo('groups:membershiprequests'),$CONFIG->wwwroot . "mod/groups/membershipreq.php?group_guid={$page_owner->getGUID()}", '1groupsactions');
+							add_submenu_item(elgg_echo('groups:membershiprequests'),$CONFIG->wwwroot . "mod/groups/membershipreq.php?group_guid={$page_owner->getGUID()}");
 					}
 					if ($page_owner->isMember($_SESSION['user'])) {
 						if ($page_owner->getOwner() != $_SESSION['guid']) {
 							$url = elgg_add_action_tokens_to_url($CONFIG->wwwroot . "action/groups/leave?group_guid=" . $page_owner->getGUID());
-							add_submenu_item(elgg_echo('groups:leave'), $url, '1groupsactions');
+							add_submenu_item(elgg_echo('groups:leave'), $url);
 						}
 					} else {
 						if ($page_owner->isPublicMembership()) {
 							$url = elgg_add_action_tokens_to_url($CONFIG->wwwroot . "action/groups/join?group_guid={$page_owner->getGUID()}");
-							add_submenu_item(elgg_echo('groups:join'), $url, '1groupsactions');
+							add_submenu_item(elgg_echo('groups:join'), $url);
 						} else {
 							$url = elgg_add_action_tokens_to_url($CONFIG->wwwroot . "action/groups/joinrequest?group_guid={$page_owner->getGUID()}");
-							add_submenu_item(elgg_echo('groups:joinrequest'), $url, '1groupsactions');
+							add_submenu_item(elgg_echo('groups:joinrequest'), $url);
 						}
 					}
 				}
 
 				if($page_owner->forum_enable != "no"){
-					add_submenu_item(elgg_echo('groups:forum'),$CONFIG->wwwroot . "pg/groups/forum/{$page_owner->getGUID()}/", '1groupslinks');
+					add_submenu_item(elgg_echo('groups:forum'),$CONFIG->wwwroot . "pg/groups/forum/{$page_owner->getGUID()}/");
 				}
 
 			}
@@ -242,12 +242,12 @@
 		// Add submenu options
 			if (get_context() == 'groups' && !($page_owner instanceof ElggGroup)) {
 				if (isloggedin()) {
-					add_submenu_item(elgg_echo('groups:new'), $CONFIG->wwwroot."pg/groups/new/", '1groupslinks');
-					add_submenu_item(elgg_echo('groups:owned'), $CONFIG->wwwroot . "pg/groups/owned/" . $_SESSION['user']->username, '1groupslinks');
-					add_submenu_item(elgg_echo('groups:yours'), $CONFIG->wwwroot . "pg/groups/member/" . $_SESSION['user']->username, '1groupslinks');
-					add_submenu_item(elgg_echo('groups:invitations'), $CONFIG->wwwroot . "pg/groups/invitations/" . $_SESSION['user']->username, '1groupslinks');
+					add_submenu_item(elgg_echo('groups:new'), $CONFIG->wwwroot."pg/groups/new/");
+					add_submenu_item(elgg_echo('groups:owned'), $CONFIG->wwwroot . "pg/groups/owned/" . $_SESSION['user']->username);
+					add_submenu_item(elgg_echo('groups:yours'), $CONFIG->wwwroot . "pg/groups/member/" . $_SESSION['user']->username);
+					add_submenu_item(elgg_echo('groups:invitations'), $CONFIG->wwwroot . "pg/groups/invitations/" . $_SESSION['user']->username);
 				}
-				add_submenu_item(elgg_echo('groups:all'), $CONFIG->wwwroot . "pg/groups/all/", '1groupslinks');
+				add_submenu_item(elgg_echo('groups:all'), $CONFIG->wwwroot . "pg/groups/all/");
 			}
 
 	}
