--- a/engine/lib/users.php
+++ b/engine/lib/users.php
@@ -1408,7 +1408,7 @@ function validate_username($username) {
 
 	// Basic, check length
 	if (!isset($CONFIG->minusername)) {
-		$CONFIG->minusername = 4;
+		$CONFIG->minusername = 2;
 	}
 
 	if (strlen($username) < $CONFIG->minusername) {
--- a/languages/en.php
+++ b/languages/en.php
@@ -398,7 +398,7 @@ To remove a widget drag it back to the <b>Widget gallery</b>.",
 
 	'registration:notemail' => 'The email address you provided does not appear to be a valid email address.',
 	'registration:userexists' => 'That username already exists',
-	'registration:usernametooshort' => 'Your username must be a minimum of 4 characters long.',
+	'registration:usernametooshort' => 'Your username must be a minimum of 2 characters long.',
 	'registration:passwordtooshort' => 'The password must be a minimum of 6 characters long.',
 	'registration:dupeemail' => 'This email address has already been registered.',
 	'registration:invalidchars' => 'Sorry, your username contains invalid characters.',
